codespell_lib/data/dictionary.txt

```diff
@@ -53286,6 +53286,8 @@ tyrany->tyranny
 tyring->trying
 tyrranies->tyrannies
 tyrrany->tyranny
+uability->usability
+uable->unable, usable,
```

Comment:
Perhaps `able`, `suable` and `table` (and even `cable`, `fable`, `gable`, `sable`).

Reply:
I'll add `table` and `able`. I couldn't find the others.